Style.css

```diff
@@ -459,6 +459,156 @@ textarea::placeholder {
   margin-top: 515px;
 }
 
+.pop-mobile-container {
+  height: 836px;
+  width: 343px;
+  left: 16px;
+  top: 20px;
+  border-radius: 16px;
+  padding-left: 16px;
+  padding-top: 16px;
+  padding-bottom: 16px;
+  color: #172b4d;
+  border: #091e42 solid 1px;
+  position: fixed;
+  bottom: 0;
+  right: 0;
+  z-index: 0;
+  overflow: auto;
+  background-color: rgb(255, 255, 255);
+  backdrop-filter: blur(10px);
+  -webkit-backdrop-filter: blur(0);
+}
+
+.mobile-pop-title {
+  height: 135px;
+  width: 259px;
+  left: 0;
+  top: 0;
+  border-radius: nullpx;
+  font-family: Inter, sans-serif;
+  font-size: 32px;
+  font-weight: 600;
+  line-height: 44px;
+  letter-spacing: 0;
+  text-align: left;
+}
+
+.prog-li-pop {
+  background-color: #ebecf0;
+  border-radius: 4px;
+  font-style: normal;
+  color: #344563;
+  margin: auto;
+  font-family: Inter, sans-serif;
+  font-size: 12px;
+  font-weight: 600;
+  line-height: 16px;
+  letter-spacing: 0.03em;
+  margin-left: 0;
+}
+
+.mobile-msg-pop {
+  height: 313px;
+  width: 311px;
+  left: 0;
+  top: 0;
+  font-family: Inter, sans-serif;
+  font-size: 16px;
+  font-weight: 400;
+  line-height: 24px;
+  letter-spacing: 0;
+  text-align: left;
+}
+
+.btn-pop {
+  display: flex;
+  flex-direction: row;
+  justify-content: flex-start;
+  height: 56px;
+  width: 311px;
+  border-radius: 0;
+  font-size: 15px;
+  text-align: left;
+}
+
+.pop-btn {
+  margin: 5px;
+}
+
+.pop-mobile-close {
+  position: fixed;
+  margin-top: 15px;
+  top: 15px;
+  right: 52px;
+  font-size: 24px;
+  color: white;
+}
+
+.img-pop-holder {
+  position: relative;
+}
+
+.background-pop {
+  background-image: url(./about-me-back.jpeg);
+  background-size: cover;
+  color: #091e42;
+  position: fixed;
+}
+
+.popup-d {
+  display: block;
+  position: fixed;
+  left: 16px;
+  top: 20px;
+}
```

Comment:
- [ ] As mentioned by the previous reviewer, the popup on mobile still has something things that will improve its resemblance to the Figma Template. Please Fix this. look at the Figma template once more and tweak you design